src/ui.js

```diff
@@ -302,6 +302,48 @@ const ui = {
     //console.log("un_hookup_ui", elem.id)
   },
 
+  injectSvgNode: (node, targetNode, width, height = width) => {
```

Comment:
Not sure about the name of this function. Open to suggestions.
It's used to scale SVG nodes, but it literally injects one into another.

Reply:
yeah, i had similar concerns.  it's mostly superficial, so whatever, nothing to gripe about right now